src/device/scrollCursorIntoView.ts

```diff
@@ -37,13 +38,15 @@ const scrollIntoViewIfNeeded = (el: Element | null | undefined) => {
   const y = window.scrollY + rect.y
 
   // leave a margin between the element and the viewport edge equal to half the element's height
+  // add 40px to the bottom margin to prevent the element from being hidden by the footer
```

Comment:
Thanks for this! Can you open a separate PR (at your hourly rate)? It's not essential for this issue, and I'd like to track the solution separately since it is a fix for #1827.

FYI The height of the NavBar is not always 40px. We'll want to measure the height like we measure the toolbar height.

Reply:
Pushed a new PR here: https://github.com/cybersemics/em/pull/2301